include/executor.h

```diff
@@ -76,8 +76,18 @@ namespace azure {  namespace storage_lite {
                         std::string str(std::istreambuf_iterator<char>(s.istream()), std::istreambuf_iterator<char>());
```

Comment:
Is there a reason that this is not consistent with the code below? I think str is also not used when code is not CURLE_OK.
Also, I think the two if statement can be merged.
Both are minor comments, consider this already signed-off with some minor suggestions :)

Reply:
> Is there a reason that this is not consistent with the code below? I think str is also not used when code is not CURLE_OK.

It's used in `else if` clause.

Reply:
And what do you mean `two if statement can be merged`, I don't see that'll simplify code logic.